lib/aruba/api.rb

```diff
@@ -729,6 +735,61 @@ def io_wait
       @aruba_io_wait_seconds || DEFAULT_IO_WAIT_SECONDS
     end
 
+    DEFAULT_ROOT_DIRECTORY = Dir.getwd
+
+    # The root directory of aruba
+    def root_directory
```

Comment:
Making aruba root directory configurable is not related to this PR (as I see it). I suggest you take that out in a seperate PR.